episodes/imaging-software.md

```diff
@@ -0,0 +1,486 @@
+---
+title: 'Imaging Software'
+teaching: 
+exercises: 
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- What are the different software options for viewing microscopy images?
+- How can Napari be used to view images?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Explain the pros and cons of different image visualisation tools (e.g. ImageJ, 
+Napari and proprietary options)
+- Use Napari to open images
+- Navigate the Napari viewer (pan/zoom/swapping between 2D and 3D views…)
+- Explain the main parts of the Napari user interface
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Choosing the right tool for the job
+
+Light microscopes can produce a very wide range of image data - for example:
```

Comment:
I think it's good to restrict to only light microscopes - as for example, electron microscopy / X-ray microscopy / atomic force microscopy etc... can have quite different approaches. I'm thinking especially of the day 2 material where we're talking more about how to setup an experiment + what factors to consider. I think confocal falls under 'light microscopes' - do you agree @drmatthews ?

Reply:
I'm happy with this focusing on light microscope. Maybe we could give some examples of popular submodalities that we think this will cover, so 
"Light microscopes (including confocal, flourescent, ...)"